cluster/src/main/java/org/apache/iotdb/cluster/log/manage/serializable/LogDequeSerializer.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.cluster.log.manage.serializable;
+
+import java.util.List;
+import org.apache.iotdb.cluster.log.Log;
+
+public interface LogDequeSerializer {
+
+  /**
+   * append a log
+   * @param log appended log
+   */
+  public void addLast(Log log, LogManagerMeta meta);
+
+  /**
+   * remove last log
+   * @param meta metadata
+   */
+  public void removeLast(LogManagerMeta meta);
```

Comment:
I think "removeLast" is no longer used, you may remove it too.

Reply:
I think we should just keep it may be we will use it after.